src/pgen/kh.cpp

```diff
@@ -78,6 +80,9 @@ void ProblemGenerator(MeshBlock *pmb, ParameterInput *pin) {
         const Real P = y < 0.25 ? P1 : P0;
         const Real vel = y < 0.25 ? v1 : v0;
 
+        Real gcov[4][4];
+        geom.SpacetimeMetric(0.0, x, y, 0.0, gcov);
```

Comment:
not that it matters much, but why set z to zero here? Why not use the z from the coords?